Server/src/routes/gameRoute.ts

```diff
@@ -66,17 +66,18 @@ gameRoute.get('/gameData/:year', async (req: Request, res: Response) => {
       });
     });
   } else {
-    res.send({ version: currentVersion });
+    res.send({ version: currentVersion, bowlData: [] });
   }
 });
 
 export const gameRouteDelta = express.Router();
 gameRoute.get('/gameData/delta/:year', async (req: Request, res: Response) => {
-  var version = 0;
+  var version = -1;
   if (!isEmpty(req.query.version)) {
     version = parseInt(req.query.version as string);
+    version = version == 0 ? -1 : version;
```

Comment:
good catch - do we need to fix this on the player?

Reply:
nvm

Reply:
glad you fixed this - will help us with testing for sure

Reply:
yee I was so confused why I didn't get any picks back (they have 0 version). I don't think we actually version the games/players when we import but rather when we run the update script. May be worthwhile looking into, but this fixes it for now and hopefully we'll deprecate excel importing at some point